config/deploy/prometheus-rule.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: reference-addon-pr
+  namespace: redhat-reference-addon
+  labels:
+    app.kubernetes.io/name: reference-addon
+    prometheus: reference-addon
+    role: alert-rules
+spec:
+  groups:
+  - name: reference-addon
+    rules:
+    - alert: reference-addon-sample-availability
+      expr: reference_addon_sample_availability == 0
+      for: 0m
+      labels:
+        severity: warning
```

Comment:
they are example metrics. one critical and one warning

Reply:
> Not an approver for this repo, but I would suggest squashing these commits since this ended up being a minor chore. Do you have more context for what this change accomplishes? I think as long as `RegisterMetrics` is called before the manager starts `init()` is not necessary.

registering custom metrics to global prometheus registry should be from init() https://book.kubebuilder.io/reference/metrics.html#publishing-additional-metrics